pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -1875,6 +1884,25 @@ class FixProcessor {
     }
   }
 
+  void _addFix_removeTypeName() {
+    final FunctionTypedFormalParameter functionTyped =
+        node is FunctionTypedFormalParameter
+            ? node
+            : node.getAncestor((node) => node is FunctionTypedFormalParameter);
+    if (functionTyped != null) {
+      _addReplaceEdit(rf.rangeNode(functionTyped),
+          utils.getNodeText(functionTyped.identifier));
+      _addFix(DartFixKind.REMOVE_TYPE, []);
+      return;
+    }
```

Comment:
Can we get confused if a function-typed formal parameter is *not* in a closure but includes something with a type of `dynamic`? Maybe something like:
```
void callFunction(void f(dynamic p)) {}
```